README.md

```diff
@@ -173,7 +173,10 @@ Doing so results in the final output sheet:
 
 # Supported Cell Types
 Currently, Exscalabur supports copying cells containing numeric, boolean, and string values. 
-Cells containing formulas and formula errors will not copied from the template to the output sheet.
+Exscalabur supports copying cell formulas if all referenced cells in the formula are on the same sheet, and
+there are no repeated rows between any cell references and the formula itself. Formulas may not behave as expected if this requirement is not met.
```

Comment:
So this doesn't work for repeated rows above or below the formula? So I can't sum up a column?

Reply:
so, the reason why we dont allow for formulas on repeated rows in general is because a referenced repeated row cell could be below the formula cell when it gets copied, and so we dont know what to substitute into the output formula cell

If we restrict that a formula cell must reference only cells above it, then those cells have to be within the same POI window, cause a formula in general cannot easily be accumulated (i think).

I _think_ the flexible writer approach could better handle this, but I haven't really dug deep into that rabbit hole.

So, limiting only to static cells within a non-repeated area is a simple first step as we spec out what needs to be done for supporting any excel formula

Reply:
So to clarify:

No formulas in repeated rows, even if it only references it's own row.

Formulas allowed in static cells as long as there are no references to cells below the formula and below repeated rows.

Reply:
uhh
as of this PR, formulas may not interact with repeated rows (cause things get complicated)

formulas may interact with static cells so long as there are no repeated rows between where the cell reference is and where the formula cell is

so, for:
```
cellA
<repeated row>
cellB
formula(cellA, cellB, cellC, cellD)
cellC
<repeated row>
cellD
```
this is illegal because there's a repeated row between cellA and the formula, and one between the formula and cellD